From b6d2d58a131173c3832328bb20b018d3c798fb6b Mon Sep 17 00:00:00 2001
From: Geoffroy Van Cutsem <geoffroy.vancutsem@intel.com>
Date: Fri, 31 Aug 2018 10:02:30 +0200
Subject: [PATCH 2070/2313] i915 GVT-g driver: adopt AcrnGT convention for
 GVT-g in ACRN

Use the AcrnGT convention when referring to GVT-g used in the
ACRN hypervisor (Project ACRN).

Signed-off-by: Geoffroy Van Cutsem <geoffroy.vancutsem@intel.com>
---
 drivers/gpu/drm/i915/gvt/acrn-gvt.c | 2 +-
 drivers/gpu/drm/i915/gvt/acrn-gvt.h | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/i915/gvt/acrn-gvt.c b/drivers/gpu/drm/i915/gvt/acrn-gvt.c
index 73575af..96a2079 100644
--- a/drivers/gpu/drm/i915/gvt/acrn-gvt.c
+++ b/drivers/gpu/drm/i915/gvt/acrn-gvt.c
@@ -46,7 +46,7 @@
 #include "acrn-gvt.h"
 
 MODULE_AUTHOR("Intel Corporation");
-MODULE_DESCRIPTION("ACRNGT mediated passthrough driver");
+MODULE_DESCRIPTION("AcrnGT mediated passthrough driver");
 MODULE_LICENSE("GPL");
 MODULE_VERSION("0.1");
 
diff --git a/drivers/gpu/drm/i915/gvt/acrn-gvt.h b/drivers/gpu/drm/i915/gvt/acrn-gvt.h
index 0799df2..f69e00b2 100644
--- a/drivers/gpu/drm/i915/gvt/acrn-gvt.h
+++ b/drivers/gpu/drm/i915/gvt/acrn-gvt.h
@@ -61,7 +61,7 @@ struct acrngt_hvm_params {
 
 /*
  * struct gvt_acrngt should be a single instance to share global
- * information for ACRNGT module.
+ * information for AcrnGT module.
  */
 #define GVT_MAX_VGPU_INSTANCE 15
 struct gvt_acrngt {
-- 
2.7.4

